docs/docs/areas.md

```diff
@@ -2,24 +2,24 @@
 title: "Areas"
 sidebar_position: 2
 ---
+import {Monitor} from "@phosphor-icons/react";
 
-## Introduction
 
 ![Area](area.png)
 
-Areas are a way to restrict the size of the canvas. You can create areas by using the [area tool](tools/area.md). If you click on the background you can open the area list dialog using the context menu.
+While you can directlz edit the content on the canvas, it is common that you would like to restrict the region in which you can modify the content, or that you would like ot be able to be able to quickly access a part of the document. For that, you have the <Monitor/> Area tool. As explained in the [introduction section](README.md), you might need to add the <Monitor/> area button to your toolbar. Using the [area tool](tools/area.md), you can draw rectangle anywhre on the canvas to then define the area.
 
-## Creating and deleting areas
+## More control 
+You can further define area tool by clicking again on the <Monitor/> area button. There, you can define the areas's fixed width, height, or aspect ratio. 
```

Comment:
please add a new line between headlines to better organize the markdown (please change also the headline below)